internal/interceptors.go

```diff
@@ -86,6 +86,7 @@ type WorkflowOutboundCallsInterceptor interface {
 	IsReplaying(ctx Context) bool
 	HasLastCompletionResult(ctx Context) bool
 	GetLastCompletionResult(ctx Context, d ...interface{}) error
+	GetLastFailure(ctx Context) error
```

Comment:
@mfateev Wanted to get your thoughts on the name of this method. Was talking with @alexshtin and he made a good point that this isn't really consistent with the rest of the SDK - should probably be `GetLastError`. However, that makes Java/Go somewhat inconsistent with each other.

Should we prefer internal consistency, or a consistency of terms? Perhaps we stay consistent internally for now, and try to be more cross-language consistent in a V2?

Reply:
I think GetLastError fits Go SDK better.

Reply:
Sounds good. Any thoughts on the overall point?